cdp_scrapers/legistar_utils.py

```diff
@@ -1432,6 +1573,32 @@ def get_events(
 
         return events
 
+    def get_updated_model(
+        self, model: IngestionModel, **kwargs: Any
+    ) -> Optional[IngestionModel]:
+        """
+        Query Legistar API for latest information for given model
+
+        Parameters
+        ----------
+        model: IngestionModel
+            e.g. ingestion_models.Matter
+
+        Returns
+        -------
+        model: Optional[IngestionModel]
+            None in case of error
+
+        See Also
+        --------
+        get_updated_matter()
+        """
+        if isinstance(model, Matter):
+            return self.get_updated_matter(model)
```

Comment:
Because I assume we are going to have a lot of these, it may be better to have a LUT.

```python
MODEL_UPDATER_LUT = {
    Matter: self.get_updated_matter,
    Person: self.get_updated_person,
}

return MODEL_UPDATER_LUT[type(model)]
```